### PR TITLE
Initial module implementation

### PR DESCRIPTION
**Pull request by glatosinski**
This pull request contains an initial implementation of the vtr_xml_utils module, with conversion tool that modifies the XML files according to the XSL files, and basic tests derived from https://github.com/SymbiFlow/symbiflow-arch-defs

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -0,0 +1,28 @@
+[tox]
+envlist = py{27,34,35,36,37}
+
+[testenv]
+setenv =
+    PYTHONPATH={toxinidir}/v2x
```
Comment (mithro) [inline tox.ini]: This path looks wrong?

**Review by mithro — APPROVED** [review]
Just one minor comment.

File: tox.ini (hunk 1 above)
Comment (glatosinski) [reply tox.ini]: Yes, I fixed this

**Review by kgugala — APPROVED** [review]
LGTM

**Pull request merged**